criticalinfrastructure/hsip-api.go

```diff
@@ -32,53 +32,74 @@ const (
+	GeneratingUnits       //https://services1.arcgis.com/Hp6G80Pky0om7QvQ/arcgis/rest/services/GeneratingUnits1/FeatureServer/0/query?outFields=*&where=1%3D1&f=geojson
 	// Hospitals
-	HurricaneEvacuationRoutes                      //https://services1.arcgis.com/Hp6G80Pky0om7QvQ/arcgis/rest/services/Hurricane_Evacuation_Routes/FeatureServer/0/query?outFields=*&where=1%3D1&f=geojson
-	IntermodalFreightFacilitiesAirtoTruck          //https://geo.dot.gov/server/rest/services/Hosted/Intermodal_Freight_Facilities_Air_to_Truck_DS/FeatureServer/0/query?outFields=*&where=1%3D1&f=geojson
-	IntermodalFreightFacilitiesMarineRollonRolloff //https://geo.dot.gov/server/rest/services/Hosted/Intermodal_Freight_Facilities_Marine_Roll_on_Roll_off_DS/FeatureServer/0/query?outFields=*&where=1%3D1&f=geojson
-	IntermodalFreightFacilitiesRail_TOFC_COFC      //https://geo.dot.gov/server/rest/services/Hosted/Intermodal_Freight_Facilities_Rail_TOFC_COFC_DS/FeatureServer/0/query?outFields=*&where=1%3D1&f=geojson
-	LandMobileBroadcastTowers                      //https://services1.arcgis.com/Hp6G80Pky0om7QvQ/arcgis/rest/services/Land_Mobile_Broadcast_Towers/FeatureServer/0/query?outFields=*&where=1%3D1&f=geojson
-	LandMobileCommercialTransmissionTowers         //https://services1.arcgis.com/Hp6G80Pky0om7QvQ/arcgis/rest/services/Land_Mobile_Commercial_Towers/FeatureServer/0/query?outFields=*&where=1%3D1&f=geojson
-	LocalEmergencyOperationsCenterEOC              //https://services1.arcgis.com/Hp6G80Pky0om7QvQ/arcgis/rest/services/Land_Mobile_Commercial_Towers/FeatureServer/0/query?outFields=*&where=1%3D1&f=geojson
-	LocalLawEnforcementLocations                   //https://services1.arcgis.com/Hp6G80Pky0om7QvQ/arcgis/rest/services/Local_Law_Enforcement_Locations/FeatureServer/0/query?outFields=*&where=1%3D1&f=geojson
-	MicrowaveServiceTowers                         //https://services1.arcgis.com/Hp6G80Pky0om7QvQ/arcgis/rest/services/Microwave_Service_Towers_New/FeatureServer/0/query?outFields=*&where=1%3D1&f=geojson
-	NationalShelterSystemFacilities                //https://gis.fema.gov/arcgis/rest/services/NSS/FEMA_NSS/FeatureServer/5/query?outFields=*&where=1%3D1&f=geojson
-	NursingHomes                                   //https://services1.arcgis.com/Hp6G80Pky0om7QvQ/arcgis/rest/services/NursingHomes/FeatureServer/0/query?outFields=*&where=1%3D1&f=geojson
-	PagingTransmissionTowers                       //https://services1.arcgis.com/Hp6G80Pky0om7QvQ/arcgis/rest/services/Paging_Transmission_Towers/FeatureServer/0/query?outFields=*&where=1%3D1&f=geojson
-	Pharmacies                                     //https://services1.arcgis.com/Hp6G80Pky0om7QvQ/arcgis/rest/services/Paging_Transmission_Towers/FeatureServer/0/query?outFields=*&where=1%3D1&f=geojson
+	HurricaneEvacuationRoutes              //https://services1.arcgis.com/Hp6G80Pky0om7QvQ/arcgis/rest/services/Hurricane_Evacuation_Routes/FeatureServer/0/query?outFields=*&where=1%3D1&f=geojson
```

Comment:
are Hurricane Evacuation routes linear features?

Reply:
According to the JSON they are LineString.